### PR TITLE
Extend the Main Menu Credits Greek Translation

### PR DESCRIPTION
**Pull request by RaptaG**
It had more space lmao :man_facepalming:

### CODE REVIEW

**Comment by RaptaG** [comment]
Why 2 commits :skull:

**Comment by Madis0** [comment]
Fix your conflicts...

File: Mod Menu Helper/assets/modmenu/lang/el_gr.json [hunk 1]
```diff
@@ -32,15 +32,16 @@
...
   "modmenu.summaryTranslation.memoryleakfix": "Διορθώνει διαρροές μνήμης RAM\n§8✎ (χωρίς επιλογές)",
   "modmenu.summaryTranslation.modmenu": "Αυτό το μενού τροποποιήσεων\n§9✎§8 Τοποθέτηση κουμπιού, φιλτράρισμα, οργάνωση ...",
   "modmenu.summaryTranslation.morechathistory": "Αυξάνει το όριο ιστορικού συνομιλίας\n§8✎ (χωρίς επιλογές)",
   "modmenu.summaryTranslation.nofade": "Όχι οθόνης φόρτωσης σε άνοιγμα πακέτων πόρων\n§8✎ (χωρίς επιλογές)",
   "modmenu.summaryTranslation.notenoughcrashes": "Ανάφερε και παίξε αν το παιχνίδι κρασάρει\n§8✎ (μόνο εξιδικευμένες επιλογές)",
   "modmenu.summaryTranslation.optigui": "Το ορισμένο γραφικό περιβάλλον του OptiFine\n§8✎ (χωρίς επιλογές)",
   "modmenu.summaryTranslation.phosphor": "§c*§r Βελτίωση φωτός, γρηγορότερα κομμάτια κόσμων\n§8✎ (μόνο εξιδικευμένες επιλογές)",
-  "modmenu.summaryTranslation.puzzle": "Μερικές επιλογές του OptiFine\n§9✎§8 Οθόνη φόρτωσης από πακέτο πόρων, ...",
+  "modmenu.summaryTranslation.puzzle": "Μερικές επιλογές του OptiFine\n§9✎§8 Πατήστε το ▼ για ρυθμίσεις",
+  "modmenu.summaryTranslation.puzzle-base": "Κάποιες επιλογές του OptiFine\n§9✎§8 Οθόνη φόρτωσης από πακέτο πόρων, ...",
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/el_gr.json:44]: Why?

Comment (RaptaG) [reply Mod Menu Helper/assets/modmenu/lang/el_gr.json:44]: I forgor

**Pull request merged**